Podfile

```diff
@@ -11,7 +11,7 @@ end
 target 'RudderAmplitude' do
     project 'RudderAmplitude.xcodeproj'
     shared_pods
-    pod 'Amplitude', '8.8.0'
+    pod 'Amplitude', '8.19.2'
```

Comment:
I think this is not needed. Please remove this.

Reply:
done